.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: rust
+
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+
+os:
+  - linux
+  - osx
+
+env:
+  - BUILD_KIND=DEBUG
+  - BUILD_KIND=RELEASE
```

Comment:
I don't think this doesn't anything? I don't think there's any need to do separate debug and release builds for this crate, in any case.